AlembicImporter/Packages/com.unity.formats.alembic/Editor/Timeline/AlembicRecorderClipEditor.cs

```diff
@@ -156,3 +157,4 @@ TimelineAsset FindTimelineAsset()
         }
     }
 }
+#endif
```

Comment:
Good practice is to add a comment following the endif, indicating what it relates to. Here for example:
#endif // ENABLE_ALEMBIC_TIMELINE_RECORDER